tests/e2e/e2e_common.go

```diff
@@ -223,6 +223,30 @@ const (
 		"/api-tokens/authorize"
 )
 
+// test suite labels
+const (
+	TestSuiteLabelP0                   = "P0"
+	TestSuiteLabelP1                   = "P1"
+	TestSuiteLabelP2                   = "P2"
+	TestSuiteLabelSerial               = "Serial"
+	TestSuiteLabelSlow                 = "Slow"
+	TestSuiteLabelFlaky                = "Flaky"
+	TestSuiteLabelDisruptive           = "Disruptive"
+	TestSuiteLabelWCP                  = "WCP"
+	TestSuiteLabelTKG                  = "TKG"
+	TestSuiteLabelBlockVanilla         = "Block-Vanilla"
+	TestSuiteLabelFileVanilla          = "File-Vanilla"
+	TestSuiteLabelTopology             = "Topology"
+	TestSuiteLabelPreferentialTopology = "Preferential-Topology"
+	TestSuiteLabelBlockConfigSecret    = "Block-Config-Secret"
```

Comment:
just config-secret
also check if '-' is supported or not and can it complicated the usage